types/global.d.ts

```diff
@@ -23,3 +23,27 @@ declare global {
     const pg: Database;
   }
 }
+
+export interface ErrorOptions {
+  code?: number | string;
+  cause?: Error;
+}
+
+export class Error extends global.Error {
```

Comment:
It is better to have some `content` object to each error. Only `cause` may not be enough.